src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -198,45 +234,46 @@ const OScapNoneOption = ({ setProfileName }) => {
   );
 };
 
-OScapNoneOption.propTypes = {
-  setProfileName: PropTypes.any,
-};
-
-const OScapSelectOption = ({ profile_id, setProfileName, input }) => {
+const OScapSelectOption = ({
+  profile_id,
+  setProfileName,
+  input,
+}: {
+  profile_id: DistributionProfileItem;
+  setProfileName: (name: string) => void;
+  input?: string;
+}) => {
   const { getState } = useFormApi();
   const { data } = useGetOscapCustomizationsQuery({
     distribution: getState()?.values?.['release'],
     profile: profile_id,
   });
-  if (
-    input &&
-    !data?.openscap?.profile_name.toLowerCase().includes(input.toLowerCase())
-  ) {
-    return null;
+
+  if (input) {
+    const profileName = data?.openscap?.profile_name;
+    if (
+      profileName &&
+      !profileName.toLowerCase().includes(input.toLowerCase())
+    ) {
+      return null;
+    }
```

Comment:
Would just adding an interrogation mark would have sufficed here?

```
  if (
    input &&
    !data?.openscap?.profile_name?.toLowerCase().includes(input.toLowerCase())
  )
```

Reply:
its enough, thank you @lavocatt